app/components/placement_details/view.rb

```diff
@@ -50,7 +50,7 @@ def missing(first_placement: false)
       placement_nominal = first_placement ? placement_records.count + 1 : 2
 
       field_label = t("components.placement_detail.placement_#{placement_nominal}")
-      link = govuk_link_to("Enter #{field_label.downcase}", new_trainee_placements_path(trainee)) if first_placement && editable
+      link = govuk_link_to("Enter #{field_label.downcase}", new_trainee_placement_path(trainee)) if first_placement && editable
```

Comment:
There are a few of these changes because I originally added placements as a singular resource in the routes file.